template/src/components/header/index.js

```diff
@@ -10,71 +10,89 @@ import 'preact-material-components/Dialog/style.css';
 import 'preact-material-components/Drawer/style.css';
 import 'preact-material-components/List/style.css';
 import 'preact-material-components/Toolbar/style.css';
+/* eslint-disable no-unused-vars */
```

Comment:
why this exactly?

Reply:
Since this is a template I understand wanting to make an empty style.css file for the header and import it even if it is never used. Frankly I don't quite understand the benefit of using `import style from './style'` and `class={style.home}` instead of `import './style'` and `class='home'`. Maybe the webpack css loader preact-cli uses auto-prefixes classes and ids if you use them this way?

Anyway, I'd be just as happy deleting line 13 and commenting out/deleting line 14 if you prefer.

Reply:
We can just set this rule to warn instead of throwing an eslint error. That will be a lot helpful instead (imo)